### DISCUSSION TITLE
长消息，这第二段内容是不是有什么问题，貌似被反代api改过

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#76] [Q&A]
第二段开头怎么是个中文
![image](https://user-images.githubusercontent.com/31379266/233918942-cb923976-80c2-480d-ba76-77ee07299723.png)

**Comment by nikissXI** [comment]
是不是api把我的回答翻译了

**Comment by linweiyuan** [comment]
上官网看看自己发的和它回复的正不正常

**Comment by nikissXI** [comment]
官网提示Too many requests in 1 hour. Try again later.
我tmd是plus啊

**Comment by linweiyuan** [comment]
看官网它的回复有没有带中文，理论上 api 只负责转发，不主动改内容，除非为了兼容旧接口

就像送快递一样

**Reply by nikissXI** [reply]
嗯，等我能打开官网再看看。
现在反代能正常使用，就是进不去官方 = =

**Reply by linweiyuan** [reply]
`GET /conversation/{id}` 这个接口可返回对话所有内容

**Reply by nikissXI** [reply]
看来确实是chatgpt自己的锅
![image](https://user-images.githubusercontent.com/31379266/233923709-110c18b0-9e04-472a-a19f-98e4005be30b.png)
